<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionControlsWidgetClass</class>
 <widget class="QWidget" name="SessionControlsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>133</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SessionControlsWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonOpenCloseDevices">
     <property name="toolTip">
      <string>Opens or closes all of the devices in the session</string>
     </property>
     <property name="text">
      <string>Open Devices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonStartStopDevices">
     <property name="toolTip">
      <string>Starts or stops all sources in the session</string>
     </property>
     <property name="text">
      <string>Start Devices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRestartDevices">
     <property name="toolTip">
      <string>Restarts all sources in the session</string>
     </property>
     <property name="text">
      <string>Restart Devices</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::SystemReboot"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRecord">
     <property name="toolTip">
      <string>Starts recording of all sources</string>
     </property>
     <property name="text">
      <string>Start Recording</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MediaRecord"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonClip">
     <property name="toolTip">
      <string>Captures a clip of the current sources</string>
     </property>
     <property name="text">
      <string>Capture Clip</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonProperties">
     <property name="toolTip">
      <string>Opens the session's properties</string>
     </property>
     <property name="text">
      <string>Properties</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
